pkg/containerspec/containerspec.go

```diff
@@ -142,3 +153,15 @@ func (in *PaastaContainerSpec) DeepCopy() *PaastaContainerSpec {
 	in.DeepCopyInto(out)
 	return out
 }
+
+func newDefaultSpec() *PaastaContainerSpec {
+	cpu := defaultCPU.copy()
```

Comment:
why do you need to take copies here?

Reply:
I think getting address of a const in go in not allowed. So need to create copy to avoid duplication of the constants defined in this file